### PR TITLE
add verse preview to settings drawer

### PR DESCRIPTION
**Pull request by muhajirdev**
image

### CODE REVIEW

File: src/utils/verse.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```
Comment (muhajirdev) [inline src/utils/verse.ts:1]: the utils is now more > 150 LoC. but not sure how we want to split the files. Open for a discussion

File: src/components/Navbar/SettingsDrawer/QuranFontSection.tsx [hunk 2]
```diff
@@ -108,6 +111,11 @@ const QuranFontSection = () => {
   return (
     <Section>
       <Section.Title>{t('fonts.quran-font')}</Section.Title>
+      <Section.Row>
+        <div className={styles.versePreviewContainer}>
+          <VersePreview />
```
Comment (osamasayed) [inline src/components/Navbar/SettingsDrawer/QuranFontSection.tsx:116]: Tajweed preview direction is flipped on mobile for LTR languages
Screen Shot 2021-11-25 at 16 07 47

File: src/components/Navbar/SettingsDrawer/VersePreview.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import { selectQuranReaderStyles } from 'src/redux/slices/QuranReader/styles';
+import QuranReaderStyles from 'src/redux/types/QuranReaderStyles';
+import { buildQCFFontFace, isQCFFont as checkIsQCFFont } from 'src/utils/fontFaceHelper';
+import { getSampleVerse } from 'src/utils/verse';
+import { QuranFont } from 'types/QuranReader';
+import Word from 'types/Word';
+
+const SWR_SAMPLE_VERSE_KEY = 'sample-verse';
+const VersePreview = () => {
+  const quranReaderStyles = useSelector(selectQuranReaderStyles, shallowEqual) as QuranReaderStyles;
```
Comment (osamasayed) [inline src/components/Navbar/SettingsDrawer/VersePreview.tsx:17]: Do we need `as QuranReaderStyles` here?

File: src/components/Navbar/SettingsDrawer/VersePreview.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        </div>
+        <div className={styles.skeletonContainer}>
+          <Skeleton>
+            <div className={styles.skeletonPlaceholder} />
+          </Skeleton>
+        </div>
+      </>
+    );
+
+  // backend return path image image (png), instead of text in Word.text. So we're mocking the same behavior here
```
Comment (osamasayed) [inline src/components/Navbar/SettingsDrawer/VersePreview.tsx:38]: nit: BE returns the path to the png image of each word

File: src/utils/verse.ts (hunk 1 above)
Comment (osamasayed) [reply src/utils/verse.ts:1]: I think we can put it inside `utils/sampleVerse.ts`

Comment (osamasayed) [reply src/utils/verse.ts:1]: Can we remove this line?

**Review by osamasayed — APPROVED** [review]

**Pull request merged**